package.json

```diff
@@ -43,43 +43,17 @@
     "node": ">= 10.0.0",
     "npm": ">= 6.0.0"
   },
-  "jest": {
-    "transform": {
-      ".(ts|tsx)": "ts-jest"
-    },
-    "testEnvironment": "node",
-    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx|js)$",
-    "moduleFileExtensions": [
-      "ts",
-      "tsx",
-      "js"
-    ],
-    "coveragePathIgnorePatterns": [
-      "/node_modules/",
-      "/test/"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 0,
-        "functions": 0,
-        "lines": 0,
-        "statements": 0
-      }
-    },
-    "collectCoverageFrom": [
-      "src/*.{js,ts}"
-    ]
-  },
   "dependencies": {
+    "@babel/runtime": "^7.12.5",
     "@ensdomains/ens": "^0.4.5",
     "@ensdomains/resolver": "^0.2.4",
-    "@ew-did-registry/claims": "0.0.1-alpha.776.0",
-    "@ew-did-registry/did": "0.0.1-alpha.632.0",
-    "@ew-did-registry/did-document": "0.0.1-alpha.776.0",
-    "@ew-did-registry/did-ethr-resolver": "0.0.1-alpha.776.0",
-    "@ew-did-registry/did-resolver-interface": "0.0.1-alpha.776.0",
-    "@ew-did-registry/jwt": "0.0.1-alpha.676.0",
-    "@ew-did-registry/keys": "0.0.1-alpha.632.0",
+    "@ew-did-registry/claims": "prerelease",
```

Comment:
Maybe we should stick to static versions of `did-library`, cause always getting the newest version while building may break something

Reply:
I agree. I also had to pin the version of the `nats.ws` because the new version uses class properties which is not yet supported by `babel`.